packages/nextjs/services/web3/wagmiConnectors.tsx

```diff
@@ -45,13 +45,14 @@ export const appChains = configureChains(
   },
 );
 
+const walletsOptions = { chains: appChains.chains, projectId: "" };
```

Comment:
We need to add `projectId`

Reply:
Would it make sense to move this `projectId` to the scaffold.config file and document it in the readme?

Reply:
Yes !!! I Should have done it already, Tysm 🙌 will update it

Reply:
Confused about whether we should add it to README or not, I have added a similar description to Alchemy API key, above it

Reply:
I think we should have a section in the README talking about the `scaffold.config.ts` file. Maybe in another issue + PR?